.jenkins

```diff
@@ -84,6 +84,7 @@ pipeline {
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Release \
+                                -DCMAKE_CXX_CLANG_TIDY="$LLVM_DIR/bin/clang-tidy" \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                 -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
Are we targeting a specific version of clang with theses builds? It's probably a good idea to make sure that the main version of kokkos llvm actually builds kokkos.

Reply:
Precisely, we are using our Kokkos fork of the LLVM project
https://github.com/kokkos/kokkos/blob/6151884b42d2c012ca71513ef13d683f9b6ed2a7/.jenkins#L72-L75

Reply:
I meant for the `CMAKE_CXX_COMPILER`.  Should we use that one from the Kokkos for as well?

Reply:
Sorry, I figured since `-DCMAKE_CXX_CLANG_TIDY="$LLVM_DIR/bin/clang-tidy" \ ` had a specific path that `clang++` would need a specific path also.